### DISCUSSION TITLE
Tabular EOS update

### DISCUSSION THREAD

**Discussion by msbc** [PrincetonUniversity/athena#376] [Ideas]
For one of my projects I need to generalize the form of the EOS table some, and I'm wondering how important is it to make the tabular EOS backwards compatible. If I am the only one who has used the tabular EOS then I don't really see the importance in maintaining backwards compatibility.

**Comment by felker** [comment]
(now Dr.) @pdmullen was using it, but he is not allowed to comment here until he celebrates passing his defense for at least a week 😄

**Reply by pdmullen** [reply]
I'll take that vacation soon, @felker ;) As for tabular EOS, my version is also very heavily modified from what is in master.  I have no need for backwards compatibility.
